Redirect to the most recently visited content scope when returning to the application (#4159)
vivid-planet/comet: .changeset/open-falcons-smash.md

---

> "@comet/cms-admin": patch

Reviewer: This isn't a bugfix, but a new feature

```suggestion
"@comet/cms-admin": minor
```

---

> Save selected `ContentScope` to local storage to redirect the user to the most recently visited scope, if no scope is provided in the domain

Reviewer: ```suggestion
Redirect to the most recently visited content scope when returning to the application
```
